Automattic/wp-calypso
# Stats: Convert CSS grid to flexbox for the Traffic page

Related to #87742

## Proposed Changes

* add a set of CSS classes to position elements in a flexbox container
* create a more flexible display system to allow for swapping of various elements by adding a CSS class
* prevent growing variations of grid definitions

## Testing Instructions



* open live branch in one tab
* open the live branch in another tab and apply `stats/flexible-grid` feature flag to the live branch
* verify that the page looks the same and modules are properly aligned on the Traffic page
* reduce the screen to below 1080px width and check that the modules take more space and the items are more readable

## Pre-merge Checklist
